### PR TITLE
Fix typo and added personal info

### PR DESCRIPTION
**Pull request by ShykulaD**

### CODE REVIEW

File: students.md [hunk 1]
```diff
@@ -2,45 +2,45 @@
...
-9 | Iaroslav Garmash | [git](https://github.com/feroxes) | Task 7 | No production experience, self learned HTML, CSS
-10 | Olexiy Letushev | [git](https://github.com/Letushev) | Task 9 | No production experience, student of Shevchenko University, HTML, CSS self learned and JS started with this course
+9 | Iaroslav Garmash | [git](https://github.com/feroxes) | Task 7 | No production experience, self-learned HTML, CSS
+10 | Olexiy Letushev | [git](https://github.com/Letushev) | Task 9 | No production experience, student of Shevchenko University, HTML, CSS self-learned and JS started with this course
 11 | Mykhailo Hanol | [git](https://github.com/ganolmc) | Task 8 | Some freelance experience about 5 years ago, Bionics University intern, interested in learning JS more
 12 | Igor Surzhyk | [git](https://github.com/isurzhyk) | Task 7 | No experience with JS, HTML, CSS, always wanted to create products
-13 | Tania Maslivets | [git](https://github.com/Masmik) | Task 9 | GOIT, self learning, second try
+13 | Tania Maslivets | [git](https://github.com/Masmik) | Task 9 | GOIT, self-earning, second try
```
Comment (mashoshyna-test) [inline students.md]: learning => earning. typo?

File: students.md [hunk 2]
```diff
@@ -2,45 +2,45 @@
...
 14 | Viktoriia Buts | [git](https://github.com/viktoriiab) | Task 8 | Student in KPI, want to work in frontend, 3-rd try
```
Comment (AMashoshyna) [inline students.md]: learning => earning. typo?

Comment (ShykulaD) [reply students.md]: Oh, I see. I will fix

Comment (AMashoshyna) [reply students.md]: you can push one more commit to your branch and it will be added to PR automatically (just in case)

Comment (ShykulaD) [reply students.md]: Ok, I will test now. Don't merge pls. Thank you!

Comment (ShykulaD) [reply students.md]: I can't push to the branch
http://take.ms/GUxlM

Comment (ShykulaD) [reply students.md]: Looks like I have fixed. 
@AMashoshyna can you check and merge if everything is okey?

**Comment by suchov** [comment]
LGTM @AMashoshyna go ahead

**Comment by AMashoshyna** [comment]
@ShykulaD thanks!

**Pull request merged**